models/model-cards/abp-model-card.md

```diff
@@ -76,15 +76,15 @@ limitations under the License.
 ## Training Dataset:
 
 **Link:**
-* https://github.com/nv-morpheus/Morpheus/blob/branch-25.02/models/datasets/training-data/abp-sample-nvsmi-training-data.json  <br>
+* https://github.com/nv-morpheus/Morpheus/blob/branch-25.06/models/datasets/training-data/abp-sample-nvsmi-training-data.json  <br>
```

Comment:
Minor comment: These files do not get updated often (if ever). Is there anything stopping us from updating this URL to something we won't need to constantly update?  
(E.g., https://github.com/nv-morpheus/Morpheus/blob/HEAD/models/datasets/training-data/abp-sample-nvsmi-training-data.json)
(Not blocking)